### PR TITLE
RELEASE.md changes

### PR DESCRIPTION
**Pull request by av8ramit**
Updating the RELEASE.md for patch version 1.0.1

### CODE REVIEW

**Comment by gunan** [comment]
I have a couple other addition requests to this file.
Could you add the changes mentioned in https://github.com/tensorflow/tensorflow/issues/7836 and https://github.com/tensorflow/tensorflow/issues/8034#event-986058509 to the `Breaking changes to the API` section in 1.0.0 release notes? You can edit the 1.0.0 notes section in place, as these were actual changes made in 1.0.0 but forgotten to be added to the release notes.

In 1.0.1, Let's also mention:
Removed `tf.core` and `tf.python` modules from the API. These were never intended to be exposed. Please use the same objects through top-level `tf` module instead.

**Comment by av8ramit** [comment]
Updated with your changes @gunan

File: RELEASE.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# Release 1.0.1
+
+## Bug Fixes and Other Changes
+* Change GraphConstructor to not increase the version when importing, but instead take the min of all versions.
+* Google Cloud Storage fixes
```
Comment (gunan) [inline RELEASE.md]: Nit. Finish sentence with "."

File: RELEASE.md [hunk 2]
```diff
@@ -86,6 +93,8 @@ To help you upgrade your existing TensorFlow Python code to match the API change
   from the tensorflow::ops namespace to tensorflow.
 * Change arg order for `{softmax,sparse_softmax,sigmoid}_cross_entropy_with_logits` to be (labels, predictions), and force use of named args.
 * tf.nn.rnn_cell.* and most functions in tf.nn.rnn.* (with the exception of dynamic_rnn and raw_rnn) are temporarily in tf.contrib.rnn.  They will be moved back into core for TF 1.1.
+* `tf.nn.sampled_softmax_loss` and `tf.nn.nce_loss` have both changed their API such that you need to switch the `inputs, labels` to `labels, inputs` parameters.
+* The shape keyword argument of the SparseTensor constructor changes its name to dense_shape between Tensorflow 0.12 and Tensorflow 1.0.
```
Comment (gunan) [inline RELEASE.md]: Nit, `SparseTensor` (between code quotes) and `dense_shape`.

**Review by gunan — APPROVED** [review]

**Pull request merged**